classes/student.rb

```diff
@@ -1,12 +1,18 @@
 require './person'
 
 class Student < Person
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+  attr_reader :classroom
+
+  def initialize(age, name = 'Unknown')
+    super(age, name)
+  end
+
+  def classroom=(classroom)
     @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
- [ ] Well! Using the `add_student` method in the `Classroom` class it is possible to add a student to a class. However when I attempted to use this method to add a classroom to the student, I encountered an error. Please consider updating your code to ensure that it can seamlessly add a classroom to the student without any errors. This adjustment will enhance the functionality of your code. Keep up the good work! 👍🚀